ui/dashboards/src/components/Panel/Panel.test.tsx

```diff
@@ -15,14 +15,66 @@ import { screen, waitFor } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { PanelDefinition } from '@perses-dev/core';
 import { DataQueriesProvider, TimeRangeProvider, useDataQueriesContext } from '@perses-dev/plugin-system';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
 import { renderWithContext } from '../../test';
 import { VariableProvider } from '../../context';
 import { Panel, PanelProps } from './Panel';
 
+const testTheme = createTheme({
+  components: {
+    MuiButtonBase: {
+      defaultProps: {
+        disableRipple: true,
+        disableTouchRipple: true,
+      },
+    },
+    MuiButton: {
+      defaultProps: {
+        disableRipple: true,
+        disableTouchRipple: true,
+      },
+    },
+    MuiIconButton: {
+      defaultProps: {
+        disableRipple: true,
+        disableTouchRipple: true,
+      },
+    },
+  },
+  transitions: {
+    create: () => 'none',
+  },
+});
+
+jest.mock('@perses-dev/components', () => ({
+  ...jest.requireActual('@perses-dev/components'),
+  InfoTooltip: ({ children, description }: { children: React.ReactNode; description: string }): JSX.Element => (
+    <>
+      {children}
+      <div role="tooltip">{description}</div>
+    </>
+  ),
+}));
+
+jest.mock('@mui/material/CircularProgress', () => {
+  return function MockCircularProgress(props: { 'aria-label'?: string }): JSX.Element {
+    return <div aria-label={props['aria-label'] || 'loading'} />;
+  };
+});
+
 jest.mock('@perses-dev/plugin-system', () => {
   return {
     ...jest.requireActual('@perses-dev/plugin-system'),
-    useDataQueriesContext: jest.fn(() => ({ queryResults: [] })),
+    useDataQueriesContext: jest.fn(() => ({
+      queryResults: [],
+      isFetching: false,
+      errors: [],
+    })),
+    usePluginRegistry: jest.fn(() => ({
+      getPlugin: jest.fn((): { PanelComponent: () => JSX.Element } => ({
+        PanelComponent: (): JSX.Element => <div>TimeSeriesChart panel</div>,
```

Comment:
Since we are mocking the plugin, we can also mock the pluginActions to test the new feature.